sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactListCriteriaBuilder.java

```diff
@@ -118,6 +119,24 @@
 			contact.get(Contact.EXTERNAL_TOKEN));
 	}
 
+	public List<Selection<?>> getContactIndexSelections(Root<Contact> contact, ContactJoins joins) {
+
+		final List<Selection<?>> indexSelection = new ArrayList<>(getContactIndexCommonSelections(contact, joins));
+		indexSelection.addAll(
+			Arrays.asList(contact.get(AbstractDomainObject.ID), contact.get(AbstractDomainObject.CREATION_DATE), contact.get(Contact.COMPLETENESS)));
+
+		return indexSelection;
+	}
+
+	public List<Selection<?>> getContactIndexSelectionsForFusion(Root<Contact> contact, ContactJoins joins) {
```

Comment:
Fusion is not a word we're using in the system right now; if you mean merging, please use that

Reply:
This hasn't been resolved yet